src/main/java/org/reactome/addlinks/fileprocessors/HPAFileProcessor.java

```diff
@@ -0,0 +1,53 @@
+package org.reactome.addlinks.fileprocessors;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class HPAFileProcessor extends FileProcessor<List<String>>
+{
+
+    public HPAFileProcessor(String processorName)
+    {
+        super(processorName);
+    }
+
+    public HPAFileProcessor()
+    {
+        super(null);
```

Comment:
Could an empty String be passed in here instead?  Whenever possible, it's preferable to pass in an empty value instead of a null value to reduce dealing with NullPointerExceptions.

Reply:
This was mimicked from @SolomonShorser-OICR's code, so I'm not sure why null was chosen here instead of an empty string

Reply:
@SolomonShorser-OICR Can an empty String be used here instead of a null value?

Reply:
`FileProcessor` extends `CustomLoggable`. In `CustomLoggable`, it's possible to log either to a custom log file, or whatever is already being logged to (like a general application log). Passing null to the constructor means that no custom logging file will be used. Now, in most cases, the constructor receives an argument from the config file, like this: `<constructor-arg name="processorName" value="file-processors/DockblasterFileProcessor"/>...`

For some instantiation situations of some things that extend from `CustomLoggable`, it was desirable for the new instance to write to the main application log, not a custom log. And for backwards compatibility purposes, a no-arg constructor was easier. 

The custom logging code _is_ a bit weird and could probably use a cleanup, but that probably should be done in a separate issue/PR. Also probably is low-priority since it works fairly well now (despite looking weird).

Reply:
Okay, understood.

Reply:
@jweiser Looking at the code in `CustomLoggable`, it should be OK to use an empty string, if you want to. Just test it first, to make sure it works.